sys/ztimer/Kconfig

```diff
@@ -71,6 +71,7 @@ config MODULE_ZTIMER_MSEC
 choice ZTIMER_MSEC_BACKEND
     bool "Backend"
     depends on MODULE_ZTIMER_MSEC
+    default ZTIMER_MSEC_BACKEND_TIMER if MODULE_ZTIMER_USEC
```

Comment:
Ahh yes, native is fun to model.  Just create a different symbol, something like `PREFER_ZTIMER_PERIPH_TIMER` so it would look like:

```diff
diff --git a/sys/ztimer/Kconfig b/sys/ztimer/Kconfig
index 38669dd5a1..277a502191 100644
--- a/sys/ztimer/Kconfig
+++ b/sys/ztimer/Kconfig
@@ -60,6 +60,10 @@ config ZTIMER_USEC
     select MODULE_ZTIMER_USEC
     select MODULE_ZTIMER_PERIPH_TIMER
     select MODULE_ZTIMER
+    select PREFER_ZTIMER_PERIPH_TIMER
+
+config PREFER_ZTIMER_PERIPH_TIMER
+    bool
 
 config MODULE_ZTIMER_USEC
     bool
@@ -71,7 +75,7 @@ config MODULE_ZTIMER_MSEC
 choice ZTIMER_MSEC_BACKEND
     bool "Backend"
     depends on MODULE_ZTIMER_MSEC
-    default ZTIMER_MSEC_BACKEND_TIMER if MODULE_ZTIMER_USEC
+    default ZTIMER_MSEC_BACKEND_TIMER if PREFER_ZTIMER_PERIPH_TIMER
     default ZTIMER_MSEC_BACKEND_LPTIMER if HAVE_ZTIMER_PERIPH_LPTIMER
     default ZTIMER_MSEC_BACKEND_RTT if !MODULE_ZTIMER_NO_PERIPH_RTT
     default ZTIMER_MSEC_BACKEND_TIMER
@@ -99,6 +103,7 @@ config MODULE_ZTIMER_SEC
 choice
     bool "Backend"
     depends on MODULE_ZTIMER_SEC
+    default ZTIMER_SEC_BACKEND_TIMER if PREFER_ZTIMER_PERIPH_TIMER
     default ZTIMER_SEC_BACKEND_LPTIMER if HAVE_ZTIMER_PERIPH_LPTIMER
     default ZTIMER_SEC_BACKEND_RTT if !MODULE_ZTIMER_NO_PERIPH_RTT
     default ZTIMER_SEC_BACKEND_TIMER
```